clustered/client/src/main/java/org/ehcache/clustered/client/internal/service/DefaultClusteringService.java

```diff
@@ -383,23 +383,17 @@ protected boolean isStarted() {
     try {
       if (configuration.isAutoCreate()) {
         try {
-          this.entity.validateCache(cacheId, clientStoreConfiguration);
-        } catch (ClusteredTierValidationException ex) {
-          if (ex.getCause() instanceof InvalidStoreException) {
-            try {
-              this.entity.createCache(cacheId, clientStoreConfiguration);
-            } catch (TimeoutException e) {
-              throw new CachePersistenceException("Unable to create clustered tier proxy '"
-                  + cacheIdentifier.getId() + "' for entity '" + entityIdentifier
-                  + "'; create operation timed out", e);
-            }
-          } else {
-            throw ex;
+          entity.createCache(cacheId, clientStoreConfiguration);
+        } catch (ClusteredTierCreationException e) {
+          // An InvalidStoreException means the cache already exists. That's fine, the validateCache will then work
+          if (!(e.getCause() instanceof InvalidStoreException)) {
+            throw e;
           }
         }
-      } else {
-        this.entity.validateCache(cacheId, clientStoreConfiguration);
       }
+
+      entity.validateCache(cacheId, clientStoreConfiguration);
```

Comment:
I think the client either creates or validates a store config. The logic earlier was to either create or validate. I think if the cache is successfully created, there is no need to validate it again as earlier.That is the reason of failure at server during reconnect in `PassiveSyncTest`, since the server goes down client waits infinitely and thus build aborts.